docs/apache-airflow/alembic_revisions.rst

```diff
@@ -0,0 +1,332 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+
+Alembic Revisions
+=================
+
+The following table shows the list of alembic revisions
+
+
+.. list-table:: Alembic Revisions(until Airflow 2.0.0):
```

Comment:
Is this just migrations since 1.10.14?

If so 

```suggestion
.. list-table:: Alembic Revisions (from Airflow 2.0.0 onwards):
```

Reply:
Let me make these changes and modify my pull request